File: module01/module39/src/main/java/com/android/example/module01_module39/ClassAAI.java
package com.android.example.module01_module39;

public class ClassAAI {
	private 	io.reactivex.subscribers.TestSubscriber<Object> instance_var_1_0 = new io.reactivex.subscribers.TestSubscriber();
	private 	io.reactivex.subscribers.TestSubscriber<Object> instance_var_1_1 = new io.reactivex.subscribers.TestSubscriber();
	private 	io.reactivex.subscribers.TestSubscriber<Object> instance_var_1_2 = new io.reactivex.subscribers.TestSubscriber();

	public void method0(
		dagger.internal.DelegateFactory<Object> param0,
		io.reactivex.subscribers.TestSubscriber<Object> param1) throws Throwable {
	}

	public void method1(
		io.reactivex.subscribers.TestSubscriber<Object> param0,
		io.reactivex.subscribers.TestSubscriber<Object> param1,
		io.reactivex.subscribers.TestSubscriber<Object> param2) throws Throwable {
		dagger.internal.DelegateFactory<Object> local_var_2_3 = new dagger.internal.DelegateFactory();
		local_var_2_3.get();

		for (int iAb = 0; iAb < 6; iAb++) {
			if (new java.lang.Object().equals(new java.lang.Object())) {
				param0.isCancelled();

			} else {
				io.reactivex.subscribers.TestSubscriber<Object> local_var_4_0 = new io.reactivex.subscribers.TestSubscriber();
				local_var_4_0.assertNotSubscribed();

			}
			io.reactivex.subscribers.TestSubscriber<Object> local_var_3_0 = new io.reactivex.subscribers.TestSubscriber();
			local_var_3_0.assertNotSubscribed();

		}
	}

	public void method2(
		dagger.internal.DelegateFactory<Object> param0,
		io.reactivex.subscribers.TestSubscriber<Object> param1,
		io.reactivex.subscribers.TestSubscriber<Object> param2,
		io.reactivex.subscribers.TestSubscriber<Object> param3) throws Throwable {
		param0.get();

	}

	public void method3(
		dagger.internal.DelegateFactory<Object> param0,
		io.reactivex.subscribers.TestSubscriber<Object> param1) throws Throwable {
		param0.get();

		dagger.internal.DelegateFactory<Object> local_var_2_2 = new dagger.internal.DelegateFactory();
		local_var_2_2.get();

		param0.get();

	}

	public void method4(
		io.reactivex.subscribers.TestSubscriber<Object> param0) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_1 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_1.assertNotSubscribed();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_2 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_2.isCancelled();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_3 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_3.requestMore(1348072073l);

	}
}

File: module06/module083/module06/src/main/java/com/android/example/module06_module083_module06/ClassAAI.kt
package com.android.example.module06_module083_module06

@Suppress("UNUSED_PARAMETER")
class ClassAAI {
	private 	val instance_var_1_0: com.android.example.module06_module109_module1.ClassAAA = com.android.example.module06_module109_module1.ClassAAA()
	private 	val instance_var_1_1: com.android.example.module06_module083_module09.ClassAAH = com.android.example.module06_module083_module09.ClassAAH()
	private 	val instance_var_1_2: com.android.example.module06_module321_module10.ClassAAH = com.android.example.module06_module321_module10.ClassAAH()

	fun method0(
		param0: com.android.example.module06_module109_module1.ClassAAD,
		param1: com.android.example.module06_module109_module1.ClassAAD,
		param2: com.android.example.module06_module109_module1.ClassAAG
	) {
		val local_var_2_3: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_3.assertNotSubscribed()

		val local_var_2_4: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_4.requestMore(-6956942557213488132)

		for (iAb in 0..2) {
			val local_var_3_0: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
			local_var_3_0.onError(kotlin.Throwable())

			local_var_3_0.isDisposed()

			val local_var_3_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
			local_var_3_1.request(-6873761404203516042)

			val local_var_3_2: io.reactivex.observers.TestObserver<kotlin.Any> = io.reactivex.observers.TestObserver()
			local_var_3_2.onError(kotlin.Throwable())

		}
		val local_var_2_5: io.reactivex.internal.disposables.ListCompositeDisposable = io.reactivex.internal.disposables.ListCompositeDisposable()
		local_var_2_5.dispose()

	}

	fun method1(
		param0: com.android.example.module06_module109_module1.ClassAAD,
		param1: com.android.example.module06_module109_module1.ClassAAB,
		param2: com.android.example.module06_module321_module10.ClassAAD,
		param3: com.android.example.module06_module083_module09.ClassAAH
	) {
	}

	fun method2(
		param0: com.android.example.module06_module321_module10.ClassAAA
	) {
		val local_var_2_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_1.hasSubscription()

		val local_var_2_2: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_2.onError(kotlin.Throwable())

		val local_var_2_3: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_3.onNext(kotlin.Any())

	}

	fun method3(
		param0: com.android.example.module06_module321_module10.ClassAAD,
		param1: com.android.example.module06_module083_module09.ClassAAI,
		param2: com.android.example.module06_module321_module10.ClassAAE
	) {
		val local_var_2_3: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_3.isDisposed()

	}

	fun method4(
		param0: com.android.example.module06_module321_module10.ClassAAF,
		param1: com.android.example.module06_module083_module09.ClassAAJ,
		param2: com.android.example.module06_module109_module1.ClassAAF,
		param3: com.android.example.module06_module321_module10.ClassAAG
	) {
		val local_var_2_4: io.reactivex.internal.disposables.ListCompositeDisposable = io.reactivex.internal.disposables.ListCompositeDisposable()
		local_var_2_4.dispose()

	}
}
